plugins/woocommerce/client/legacy/js/frontend/address-autocomplete.js

```diff
@@ -663,7 +662,7 @@ window.wc.addressAutocomplete.registerAddressAutocompleteProvider =
 						addressData.state
 					);
 				}
-			}, 200 );
+			}, 50 );
```

Comment:
Reduced the timeout needed to fill values from select, honestly even 50 is too much, we just need to wait for states to load in this.